protobuf/lib/src/protobuf/coded_buffer_reader.dart

```diff
@@ -164,19 +164,22 @@ class CodedBufferReader {
     return _buffer[_bufferPos - 1];
   }
 
-  int _readRawVarint32([bool signed = true]) {
+  int _readRawVarint32(bool signed) {
     // Read up to 10 bytes.
-    int bytes = _currentLimit - _bufferPos;
+    int bufferPos = _bufferPos;
```

Comment:
Maybe you could leave a comment here explaining why you are saving `_bufferPos` in a local variable.

Reply:
Done.